datasets/s2orc/README.md

```diff
@@ -0,0 +1,241 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc-by-2.0
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+- text-classification
+task_ids:
+- language-modeling
+- multi-label-classification
```

Comment:
Why multi-label-classification ?

Also I think we can add
```
- other-citation-recommendation
```

Reply:
After looking at `fieldsOfStudy` field I thought we can have multi-label classification for this. Yeah, we can add citation recommendation

Reply:
It should be `other-other-citation-recommendation` I guess

Reply:
Yes you're right :)